src/group/type.ts

```diff
@@ -0,0 +1,4 @@
+export type GroupMember = {
```

Comment:
i guess this type isn't being used at all, so we could remove it

Reply:
Removed!